docs/data-ai/ai/make-decisions-autonomously.md

```diff
@@ -0,0 +1,1059 @@
+---
+linkTitle: "Make decisions autonomously"
+title: "Make decisions autonomously"
+weight: 70
+layout: "docs"
+type: "docs"
+description: "Use the vision service API to act based on inferences."
+next: "/data-ai/train/upload-external-data/"
+aliases:
+  - /data-ai/ai/act/
+---
+
+Use the [vision service API](/dev/reference/apis/services/vision/) to make inferences, then use [component APIs](/dev/reference/apis/#component-apis) to react to inferences with a machine.
+
+## Follow a line
+
+This module uses a vision service and a motor to program a machine to follow a line of a configurable color.
+
+### Prerequisites
+
+- An SBC, for example a Raspberry Pi 4
+- A wheeled base component such as a [SCUTTLE robot](https://www.scuttlerobot.org/shop/)
+- A webcam
+- Colored tape, to create a path for your robot
+
+### Configure your machine
+
+Follow the [setup guide](/operate/get-started/setup/) to create a new machine.
+
+Connect your SCUTTLE base to your SBC.
+Add the following `components` configuration to create board, base, and motor components in Viam so you can control your SCUTTLE base:
```

Comment:
I would make this way less scuttle-specific. You could link to the tutorial as an example but this feels like too much detail and too specific for an AI section guide.

I'd ditch the config below too since it takes up so much length on the page but won't apply to all bases; just tell them to configure their base with a link to https://docs.viam.com/operate/reference/components/base/